drbd_metrics.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import (
+	"encoding/json"
+	"log"
+	"os/exec"
+)
+
+// drbdStatus is for parsing relevant data we want to convert to metrics
+type drbdStatus struct {
+	Name    string `json:"name"`
+	Role    string `json:"role"`
+	Devices []struct {
+		Volume    int    `json:"volume"`
+		DiskState string `json:"disk-state"`
+	} `json:"devices"`
+}
+
+// return drbd status in byte raw json
+func getDrbdInfo() []byte {
+	// get ringStatus
+	log.Println("[INFO]: Reading drbd status with drbdsetup status ...")
+	drbdStatusRaw, _ := exec.Command("/sbin/drbdsetup", "status", "--json").Output()
```

Comment:
we could return this error and log it in the main, I'm not super comfortable in suppressing it completely; what do you think?

Reply:
ok